packages/util/src/types/index.ts

```diff
@@ -1,5 +1,8 @@
 import React from 'react'
 
+/**
+ * @deprecated
```

Comment:
Nice. Also would be a good idea to add a note with this in the future about when it happened and when we will officially remove it.

Typically, this is normally not enough since it's really just an IDE feature. Would be better to rename it using the common convention of 
```
RefFor_DEPRECATED
```

However, no need to do it now. Just future reference.